apps/fifty-fifty-php/app/Http/Middleware/EncryptCookies.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Illuminate\Cookie\Middleware\EncryptCookies as Middleware;
+
+class EncryptCookies extends Middleware
+{
+    /**
+     * The names of the cookies that should not be encrypted.
+     *
+     * @var array<int, string>
+     */
+    protected $except = [
+        //
```

Comment:
The same note about the empty comment. Now I start to think that maybe they're used as some placeholders (?)